k8s/kube-base/deployment-mariadb.yaml

```diff
@@ -25,7 +25,34 @@ spec:
           volumeMounts:
             - name: vol-mariadb
               mountPath: /var/lib/mysql
+            - name: vol-my-conf
+              mountPath: /etc/mysql/my.cnf
```

Comment:
In the mariadb image, `/etc/mysql/my.cnf` is a symlink (to `/etc/mysql/mariadb.cnf`), so doesn't need to be replaced.

```
root@807f537634eb:/etc/mysql# ls -l /etc/mysql/my.cnf
lrwxrwxrwx. 1 root root 24 Aug  2 20:21 /etc/mysql/my.cnf -> /etc/alternatives/my.cnf
root@807f537634eb:/etc/mysql# ls -l /etc/alternatives/my.cnf
lrwxrwxrwx. 1 root root 22 Aug  2 20:21 /etc/alternatives/my.cnf -> /etc/mysql/mariadb.cnf
root@807f537634eb:/etc/mysql#

Reply:
Hold on, I might be lying about this. Let me make sure I'm looking at the same image.

Reply:
Never mind, this is a difference between the `mariadb:10` image (which gets us 10.8.3) and the explicit  10.3.34 you're using in the config.

Reply:
Not certain which container you are looking at here.  For example, when I deploy from the ArgoCD set of yaml files, running your commands, I get:

        $ ls -la /etc/mysql/my.cnf
        -rw-r--r--. 1 root root 5221 Apr 30 01:20 /etc/mysql/my.cnf
        $ ls -la /etc/mysql/mariadb.cnf 
        -rw-r--r--. 1 root root 527 Feb 10  2022 /etc/mysql/mariadb.cnf

When gives 2 different files.  Furthermore, the my.cnf file that the following that loads mariadb.cnf with the following:

        #
        # * IMPORTANT: Additional settings that can override those from this file!
        #   The files must end with '.cnf', otherwise they'll be ignored.
        #
        !include /etc/mysql/mariadb.cnf

And loads more with the following (at the end of the my.cnf file):

        # Import all .cnf files from configuration directory
        !includedir /etc/mysql/mariadb.conf.d/
        
Perhaps there is some way that we can reconcile what you are seeing with what I am seeing?